compiler/luci/export/src/CircleExporterUtils.cpp

```diff
@@ -88,15 +88,18 @@ circle::MirrorPadMode to_circle_mirrorpadmode(luci::MirrorPadMode mode)
 namespace luci
 {
 
-uint32_t SerializedModelData::registerBuiltinOpcode(circle::BuiltinOperator builtin_code)
+uint32_t SerializedModelData::registerBuiltinOpcode(circle::BuiltinOperator builtin_code,
+                                                    const int32_t op_version = 0)
```

Comment:
please do not use default value `= 0`

Reply:
I will remove default value to prevent the case that someone forgot to put the value :)